homeassistant/components/apsystems/strings.json

```diff
@@ -72,5 +72,10 @@
         "name": "Inverter status"
       }
     }
+  },
+  "exceptions": {
+    "inverter_error": {
+      "message": "Inverter error."
```

Comment:
```suggestion
      "message": "Inverter returned an error"
```

Reply:
I think it would be better to include the error details, maybe as a placeholder, but it looks like that is currently hidden/swallowed by the library